app/Rules/Uppercase.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class Uppercase implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     *
+     * @param  \Closure(string): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        $firstLetterUpper = strtoupper($value[0]) . substr($value, 1);
```

Comment:
You can use the `ucfirst` function.

see also: https://www.php.net/manual/en/function.ucfirst.php